src/freertos_drivers/sources

```diff
@@ -51,6 +52,7 @@ endif
 ifeq ($(TARGET),freertos.armv7m.exc)
 
 SUBDIRS += stm32cubef303x_28x_58x_98x stm32cubef303xe stm32cubef767xx \
+         spiffs_stm32f303xe spiffs_stm32f767xx \
```

Comment:
Alignment with line above, add two spaces to the indent.

Reply:
done